source/monitoring.txt

```diff
@@ -200,9 +342,22 @@ Sink Connector JMX Metrics
      - The total number of failed batch writes attempted by the MongoDB sink
        task with a duration that exceeded 10000 milliseconds.
 
+.. _kafka-monitoring-source-attributes:
+
 Source Connector JMX Metrics
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. note:: Source Connector Tasks and getMore
```

Comment:
Issue: I don't have as much context in this area, but the following paragraph seems to talk about two subsections of Source Connector Tasks: initial commands and `getMore` commands.

```suggestion
.. note:: Initial Commands and getMore Commands
```

Reply:
Good suggestion, this is a better title for the note